internal/version/version.go

```diff
@@ -51,20 +51,28 @@ type Info struct {
 	Release    bool      `json:"release,omitempty"`
 }
 
-// SemVer represents a semantic version
+// SemVer represents a semantic version.
 // TODO: replace with semver.Version
 type SemVer struct {
 	semver.Version
 }
 
-// AllowedVersions provides the Minimum SemVer per Service
+// OldSemVer represents a semantic version.
+// NB: this will be deprecated in favor of `SemVer`; these structs marshal to JSON differently.
+type OldSemVer struct {
+	Major int64 `json:"major"`
+	Minor int64 `json:"minor"`
+	Patch int64 `json:"patch"`
+}
+
+// AllowedVersions provides the Minimum SemVer per Service.
 // TODO: I don't think this name is representative of what this struct now holds.
 type AllowedVersions struct {
-	Satellite   SemVer
-	Storagenode SemVer
-	Uplink      SemVer
-	Gateway     SemVer
-	Identity    SemVer
+	Satellite   OldSemVer
+	Storagenode OldSemVer
+	Uplink      OldSemVer
+	Gateway     OldSemVer
+	Identity    OldSemVer
```

Comment:
Did we drop Bootstrap because it was part of Kademlia?

Reply:
Yea, I think that was merge before I made my changes.